parts/kubernetes/agentpool/kubelet.service

```diff
@@ -0,0 +1,53 @@
+[Unit]
```

Comment:
So we intentionally decided we wanted to keep the agent pool definitions separate from the rest of `/parts` to remove complexity and so we could have cleaner definitions.

The logic here is that this unit file is different than the rest of the kubernetes work, so we wanted to keep it separated.

Also this API is coupled with the parts directory `/parts/kubernetes/agentpool` so it will include those definitions and *only* those definitions while parsing. That way we can just define the pieces here explicitly and there is no need for logic in them.

Also, no switch flipping necessary - just use the other API and this is what you get